From 599cd1751cae649ae650333da4866f6565f21f46 Mon Sep 17 00:00:00 2001
From: Daniel Caires <DanielMarques.Caires@windriver.com>
Date: Fri, 17 Jan 2025 14:33:55 -0300
Subject: [PATCH] Change Ingress chart name

When activating the new Ingress chart in STX-openstack
it was noted that because it had the same name as the
Ingress used in the platform, the new Ingress in STX-O
could not be properly configured.

This patch changes the name of the Ingress Helm Chart,
so it differs from the one used by the platform.

Signed-off-by: Daniel Caires <DanielMarques.Caires@windriver.com>
[ Updated maintainers and version ]
Signed-off-by: Mateus Nascimento <mateus.soaresdonascimento@windriver.com>
[ Updated version ]
Signed-off-by: Vinicius Lobo <vinicius.rochalobo@windriver.com>
---
 charts/ingress-nginx/Chart.yaml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/charts/ingress-nginx/Chart.yaml b/charts/ingress-nginx/Chart.yaml
index 3ee839a..31ed0a0 100644
--- a/charts/ingress-nginx/Chart.yaml
+++ b/charts/ingress-nginx/Chart.yaml
@@ -17,7 +17,7 @@ maintainers:
 - name: Gacko
 - name: strongjz
 - name: tao12345666333
-name: ingress-nginx
+name: ingress-nginx-openstack
 sources:
 - https://github.com/kubernetes/ingress-nginx
 version: 4.12.1
-- 
2.34.1

